<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>581</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="projectDirectoryLabel">
      <property name="text">
       <string>Project Directory</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="projectRoot"/>
    </item>
    <item row="5" column="1">
     <widget class="QWidget" name="projectDirectoryWidget" native="true"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="projectDirectoryLabel_2">
      <property name="text">
       <string>Template Directory</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="templatePath"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="projectDirectoryLabel_3">
      <property name="text">
       <string>nextQuoteNumber</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="projectDirectoryLabel_4">
      <property name="text">
       <string>nextInvoiceNumber</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="nextQuoteNumber"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="nextInvoiceNumber"/>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>430</y>
     <width>549</width>
     <height>36</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>601</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Installer</string>
   </property>
   <widget class="QLineEdit" name="installer_company">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>414</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="projectDirectoryLabel_5">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>30</y>
      <width>134</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Company</string>
    </property>
   </widget>
   <widget class="QLabel" name="projectDirectoryLabel_6">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>60</y>
      <width>134</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>First/Last-Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="installer_firstName">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="installer_lastName">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="projectDirectoryLabel_8">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>90</y>
      <width>134</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Street / Nr</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="installer_street">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>271</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="projectDirectoryLabel_9">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>120</y>
      <width>134</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Zip / City</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="installer_zip">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="installer_city">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="projectDirectoryLabel_11">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>150</y>
      <width>134</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Phone</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="installer_phone">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>414</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="installer_streetNumber">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="projectDirectoryLabel_12">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>180</y>
      <width>134</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>E-Mail</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="installer_email">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>414</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>projectRoot</tabstop>
  <tabstop>templatePath</tabstop>
  <tabstop>nextQuoteNumber</tabstop>
  <tabstop>nextInvoiceNumber</tabstop>
  <tabstop>installer_company</tabstop>
  <tabstop>installer_firstName</tabstop>
  <tabstop>installer_lastName</tabstop>
  <tabstop>installer_street</tabstop>
  <tabstop>installer_streetNumber</tabstop>
  <tabstop>installer_zip</tabstop>
  <tabstop>installer_city</tabstop>
  <tabstop>installer_phone</tabstop>
  <tabstop>installer_email</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
